lib/access-code-manager.js

```diff
@@ -22,6 +22,37 @@ module.exports = (config, promise, random, storageServiceFactory) => {
     var codeLength, secretLength, storageService;
 
     /**
+     * Verifies that an account ID is not empty.
+     *
+     * @param {string} accountId
+     * @return {Promise.<*>}
+     */
+    function checkAccountIdAsync(accountId) {
+        if (accountId) {
+            return promise.resolve();
+        }
+
+        return promise.reject(new Error("Account ID must not be empty"));
+    }
+
+
+    /**
+     * Verifies an account ID and a code are not empty.
+     *
+     * @param {string} accountId
+     * @param {string} code
+     * @return {Promise.<*>}
+     */
+    function checkBothAsync(accountId, code) {
```

Comment:
This is rewritten in session-manager, with the only difference being the name of the parameter and the error message. Given that it's only written twice, it doesn't seem totally necessary to abstract it out of here and session-manager, unless it seems likely that it will be written a third time down the line. Cross that bridge when we get to it, perhaps?